### PR TITLE
fix multiple null pointer

### PR DESCRIPTION
**Pull request by jgj212**
fix multiple null pointer

### CODE REVIEW

File: coders/art.c [hunk 1]
```diff
@@ -325,6 +325,8 @@ static MagickBooleanType WriteARTImage(const ImageInfo *image_info,Image *image,
   (void) WriteBlobLSBShort(image,0);
   (void) WriteBlobLSBShort(image,(unsigned short) image->rows);
   quantum_info=AcquireQuantumInfo(image_info,image);
+  if(quantum_info==NULL)
```
Comment (dlemstra) [inline coders/art.c]: Could you change this to: `if (quantum_info == (QuantumInfo *) NULL)`? And also include the type for all the other checks?

Comment (jgj212) [reply coders/art.c]: Ok, i will change all of them

**Comment by dlemstra** [comment]
Thanks for fixing this for us!

**Comment by dlemstra** [comment]
Have you checked if we also have these issues in IM6? Could you create a new pull request to fix the mistakes in IM6?

**Pull request merged**